tests/test_planet_routes.py

```diff
@@ -0,0 +1,104 @@
+def test_get_one_planet_succeeds(client, two_saved_planets):
+    response = client.get("planets/1")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == {
+        "id": 1,
+        "name": "Mercury",
+        "description": "similar size to Earth's moon",
+        "num_of_moons": 0
+    }
+
+
+def test_get_one_planet_not_found_in_empty_db(client):
+    response = client.get("planets/1")
+    response_body = response.get_json()
+
+    assert response.status_code == 404
+    assert response_body == {"message": f"Planet 1 not found"}
```

Comment:
Small note here! The f-string you are using as the message is not necessary! It is true that the response we are returning from our route uses and f-string, but in the response itself, it it just a string. In fact, f-strings are just strings that we format differently. We would only need to use an f-string in this assert if we were bringing in a value. Because we aren't, you should be able to remove the f and it will still work!